addons/website_mass_mailing/i18n/website_mass_mailing.pot

```diff
@@ -108,6 +108,13 @@ msgstr ""
 msgid "Display Name"
 msgstr ""
 
+#. module: website_mass_mailing
+#. openerp-web
+#: code:addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js:0
+#, python-format
```

Comment:
To update .pot file, you should:
- Install the module with your branch
- Go to settings -> translation -> export translation
- Export the pot file by choosing "empty translation" (something like that) + PO type
- Replace the content of the git .pot file
- Only update what's related to your branch (there are probably many mistakes made by other but better not to noise our fixes PR)

Here I think you made a mistake doing it by hand, but I may be wrong?

Reply:
Yes BSO had given me this useful tip few weeks ago, I double checked and it's ok :+1:

Reply:
Ok, I thought the `#, python-format` stuff was not added in this case, my bad then :grimacing: